app/controllers/ems_common.rb

```diff
@@ -77,6 +77,12 @@ def display_hosts
     nested_list('hosts', Host, :breadcrumb_title => _("Managed Hosts"))
   end
 
+  def view_setup_params
+    {
+      'physical_servers' => [PhysicalServer, _("Physical Servers")],
+    }
+  end
```

Comment:
I don't see this being called anywhere. Probably something we removed when cleaning up the nested lists screens.